src/content/settings/improve-accessibility-insights.tsx

```diff
@@ -11,7 +11,11 @@ export const telemetryPopupCheckboxTitle = `I agree to enable telemetry`;
 
 export const telemetryNotice = (
     <>
-        <p>By opting into telemetry, you help the community develop inclusive software.</p>
+        <p>
+            By opting into telemetry, you{' '}
+            <NewTabLink href="https://accessibilityinsights.io/docs/en/telemetry">help the community</NewTabLink> develop inclusive
```

Comment:
@ferBonnin / @risanaka is `https://accessibilityinsights.io/docs/en/telemetry` a sufficiently stable link to encode in the product like this, or should we be using a fwlink here?

Reply:
This came from the bug description. I think we're fine for now.

Reply:
The url came from the bug description/discussion between PM's. I think we're fine for now.